docs/references/vod/patch.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: PATCH updating storage
+---
+
+`PATCH /asset/:id -d { }` can be used to modify any `asset` that was uploaded with On Demand. It is a single endpoint that can update only the following mutable properties of an asset:
+
+- `name`
+- `storage`
+- `meta`
+
+## Update Stored Asset
+
+## Updating Asset Name
+
+This will allow the name of the stored asset in Livepeer Studio to be changed.
+
+### Request
+
+```bash
+curl -X PATCH https://livepeer.studio/api/asset/{id} \
+-H 'content-type: application/json' \
+-H 'authorization: Bearer {api-key}' \
+-d { "name": "New name" }
+```
+
+## Updating Storage
+
+This allows to add additional storage location of where the assets are to be stored.
+
+> **NOTE:**
+
+- Currently assets cannot be deleted from the default Livepeer Studio storage. You can customize any additional storage though.
+- Only IPFS is supported as an additional storge at the moment. We are currently working on support for other decentralized storages like Arweave and Storj.
+
+### Request
+
+```bash
+curl -X PATCH https://livepeer.studio/api/asset/{id} \
+-H 'content-type: application/json' \
+-H 'authorization: Bearer {api-key}' \
+-d {"storage": {"ipfs": {}}}
+```
+
+## Updating Metadata
+
+This allows for modifying the metadata of the asset, which can also be customizable for NFTs. Since each marketplace has their own standard for the `metatdata` of the NFT, it is up to you to determine which standard to use and fields to include.
+
+> **NOTE:**
+
+- Livepeer Studio's defaut fields for `metadata` are:
+
+```json
+    "name": , //Populated with the asset name
+    "description": , //A default description referring to the asset name above
+    "image": //A default image including the Livepeer logo as a thumbnail
+    "animation_url": //URL for the Livepeer player on IPFS playing the video from Livepeer Studio
+    "external_url": //URL for the Livepeer player hosted on lvpr.tv playing the video from Livepeer Studio
+    "properties": {
+      "video": //CID of the original video file saved on IPFS
+      "com.livepeer.playbackId": //playback ID of the asset in Livepeer Studio
+    }
+```
+
+> **NOTICE** that you can change the value or delete any of these default fields by providing a value in the `nftMetadata` field. You can delete any of the above fields by specifying them as `null` in your overrides.
+
+Example: `"animation_url": null`
+
+### Request
+
+```bash
+curl -X PATCH https://livepeer.studio/api/asset/{id} \
+-H 'content-type: application/json' \
+-H 'authorization: Bearer {api-key}' \
+-d {
+    "storage": {
+    "ipfs": {
+        "spec": {
+            "nftMetadata": {
+                "description": "New description of asset",
+                "name": "New Name"
+            }
+        }
+    }
+  }
+}
+```
+
+## Updating Multiple parameters
```

Comment:
Do you know why the PR is not showing a diff between the old and new versions? It's showing as if you created an entire new file (and didn't delete any other), which gets a little harder to review.

Reply:
I'm not sure, I was wondering about that when I made the PR
I think it is here under the `New changes` listed above
https://github.com/livepeer/livepeer-studio-docs/pull/167/files/c07312fd08d43ef507431e4074d9d2ba48eb95f3..d31e15c5740d61d3eb504d6538762d4a2ecf21fb